rest-api/offline/sa_key_remove.py

```diff
@@ -0,0 +1,76 @@
+from googleapiclient import discovery
+import os
+import logging
+from logging.handlers import RotatingFileHandler
+from gcp import get_key, get_projects
+from datetime import datetime
+from dateutil.relativedelta import relativedelta
+
+"""Deletes service account keys older than 3 days as required by NIH"""
+_DELETE_PREFIX = 'awardee-'
+
+if os.path.isfile(get_key()):
+  os.environ["GOOGLE_APPLICATION_CREDENTIALS"] = get_key()
+
+alert = False
+
+path = os.path.expanduser('~/python-logs')
+logfile = os.path.expanduser('~/python-logs/security.log')
+
+if os.path.isdir(path):
+  pass
+else:
+  os.mkdir(path)
+
+
+logger = logging.getLogger("Rotating Log")
+log_formatter = logging.Formatter('%(asctime)s\t %(levelname)s %(message)s')
+logger.setLevel(logging.INFO)
+handler = RotatingFileHandler(logfile, maxBytes=5*1024*1024, backupCount=5)
+handler.setFormatter(log_formatter)
+logger.addHandler(handler)
+
+logger.info('-----Checking Service Account Key age-----')
```

Comment:
As discussed offline:

- Delete the logic above, and wrap the logic from here on down in a function
- Use app_identity.get_application_id() to get the project name rather than listing projects
- Call the function from a new endpoint defined in offline/main.yaml
- Add something to the cron YAML files for hitting the endpoint
- Add a file to rdr_client for invoking the endpoint manually, and test it after deploying to test or sandbox

Reply:
done